### PR TITLE
Add support for gz.msgs.EntityWrench

### LINKED ISSUE #566: ros_gz_bridge support for gz.msgs.EntityWrench
Hello,
I would like to apply external forces to robot links in Gazebo using the [apply_link_wrench](https://github.com/gazebosim/gz-sim/tree/gz-sim8/src/systems/apply_link_wrench) plugin and exposing its corresponding Gazebo topics to ROS topics.

However, apply_link_wrench uses the gz.msgs.EntityWrench message, which is not included in the correspondence table of [ros_gz_bridge](https://github.com/gazebosim/ros_gz/tree/ros2/ros_gz_bridge).

## Desired behavior

Implement a corresponding ROS message type (something like ros_gz_interfaces/msg/EntityWrench) and a bridge mapping between it and gz.msgs.EntityWrench.

## Alternatives considered

Considered using the [apply_joint_force](https://github.com/gazebosim/gz-sim/tree/gz-sim8/src/systems/apply_joint_force) plugin instead, as it uses the gz.msgs.Double type, which is included in ros_gz_bridge. Specifically for my use case it would be enough, however loading this plugin stops the controller (ros2_control) from working. In contrast, the apply_link_wrench plugin can be loaded (attached to the world) without interfering with the controller.

## Implementation suggestion

The new message type can be similar to the already implemented ros_gz_interfaces/msg/JointWrench as follows:

```bash
std_msgs/Header header
std_msgs/String entity
geometry_msgs/Wrench wrench
```
Suggested fields correspond to the [gz.msgs.EntityWrench](https://github.com/gazebosim/gz-msgs/blob/gz-msgs10/proto/gz/msgs/entity_wrench.proto) message, enabling ros_gz_bridge to convert between them.

## Additional context

As the [apply_body_wrench service](https://classic.gazebosim.org/tutorials?tut=ros_comm&cat=connect_ros#ApplyWrenchestoLinks) that was available in Gazebo Classic seems to have no direct replacement in modern Gazebo, I believe the suggested interface together with the already available apply_link_wrench public is a welcome feature.

If all this makes sense, I would be pleased to help with the implementation. Thanks in advance!

### PR DESCRIPTION
**Pull request by Vtn21**
# 🎉 New feature

Closes #566

## Summary

Adds the new ros_gz_interfaces/msg/EntityWrench, with structure compatible with gz.msgs.EntityWrench. This enables the user to apply wrenches to Gazebo entities using ROS topics.

Also implements the ros_gz_bridge mapping between ros_gz_interfaces/msg/EntityWrench and gz.msgs.EntityWrench.

## Test it

Manual testing involves spawning any entity in Gazebo and loading the gz::sim::systems::ApplyLinkWrench plugin to the world SDF. Then, call the ros_gz_bridge to expose the entity wrench topic as follows (YAML syntax):

```yaml
- topic_name: "/world/<world_name>/wrench"
  ros_type_name: "ros_gz_interfaces/msg/EntityWrench"
  gz_type_name: "gz.msgs.EntityWrench"
```

Wrenches are then applied to selected entities using the "/world/<world_name>/wrench" topic.

## Checklist
- [x] Signed all commits for DCO
- [x] Added tests
- [ ] Added example and/or tutorial
- [x] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [x] Consider updating Python bindings (if the library has them)
- [x] `codecheck` passed (See [contributing](https://gazebosim.org/docs/all/contributing#contributing-code))
- [x] All tests passed (See [test coverage](https://gazebosim.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Agazebosim+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

**Review by ahcorde — CHANGES_REQUESTED** [review]
Can you retarget the PR to `rolling` ? then we can backport this to other branches.

We use mergify to backport PR, there is no need to create a PR for each branch

File: ros_gz_bridge/README.md [hunk 1]
```diff
@@ -5,72 +5,68 @@ between ROS and Gazebo Transport.
...
-| std_msgs/msg/UInt32                            | gz.msgs.UInt32                      |
-| tf2_msgs/msg/TFMessage                         | gz.msgs.Pose_V                      |
-| trajectory_msgs/msg/JointTrajectory            | gz.msgs.JointTrajectory             |
-| vision_msgs/msg/Detection2D                    | gz.msgs.AnnotatedAxisAligned2DBox   |
-| vision_msgs/msg/Detection2DArray               | gz.msgs.AnnotatedAxisAligned2DBox_V |
-| vision_msgs/msg/Detection3D                    | gz::msgs::AnnotatedOriented3DBox    |
-| vision_msgs/msg/Detection3DArray               | gz::msgs::AnnotatedOriented3DBox_V  |
+| ROS type                                    | Gazebo type                                 |
+|---------------------------------------------|:-------------------------------------------:|
+| builtin_interfaces/msg/Time                 | ignition::msgs::Time                        |
```
Comment (ahcorde) [inline ros_gz_bridge/README.md]:
```suggestion
| builtin_interfaces/msg/Time                 | gz::msgs::Time                        |
```
Please fix the table

**Review by ahcorde — CHANGES_REQUESTED** [review]

File: ros_gz_bridge/README.md [hunk 2]
```diff
@@ -5,72 +5,68 @@ between ROS and Gazebo Transport.
...
-| std_msgs/msg/UInt32                            | gz.msgs.UInt32                      |
-| tf2_msgs/msg/TFMessage                         | gz.msgs.Pose_V                      |
-| trajectory_msgs/msg/JointTrajectory            | gz.msgs.JointTrajectory             |
-| vision_msgs/msg/Detection2D                    | gz.msgs.AnnotatedAxisAligned2DBox   |
-| vision_msgs/msg/Detection2DArray               | gz.msgs.AnnotatedAxisAligned2DBox_V |
-| vision_msgs/msg/Detection3D                    | gz::msgs::AnnotatedOriented3DBox    |
-| vision_msgs/msg/Detection3DArray               | gz::msgs::AnnotatedOriented3DBox_V  |
+| ROS type                                    | Gazebo type                                 |
+|---------------------------------------------|:-------------------------------------------:|
+| builtin_interfaces/msg/Time                 | gz::msgs::Time                        |
```
Comment (ahcorde) [inline ros_gz_bridge/README.md:10]: Not only this one, the rest of item too

**Review by ahcorde — CHANGES_REQUESTED** [review]

**Comment by Vtn21** [comment]
Hi @ahcorde , sorry for my delay. While working on merging the PR with the base ```ros2``` branch, I ended up forgetting to signoff a commit and things got messy. I opened another PR checking out directly from the ```ros2``` branch which I believe is better now. Thanks for the support!

**Pull request closed without merging**